src/commands/log.js

```diff
@@ -22,6 +22,9 @@ class LogCommand extends Command {
       await log.getFullLog()
     } catch (error) {
       spinner.fail(`Could not fetch the log "${logId}". Error: ${error.statusCode}`)
+      if (error.statusCode == 403) {
+        this.log(`${chalk.red.bold('Authentication Error')}: Log out and try again`)
+      }
```

Comment:
I think it's probably better to return this instead of the default error message. This would result in two errors being displayed. Example:

```suggestion
      const errorMessage = error.statusCode === 403
        ? 'Authentication error, try logging out and back in again.'
        : `Could not fetch the log "${logId}". Error: ${error.statusCode}`;
      spinner.fail(errorMessage)
```

What do you think?

Reply:
Sure, we can report the error in the same error message.